ttnn/cpp/ttnn/operations/data_movement/slice/slice_pybind.hpp

```diff
@@ -0,0 +1,63 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "slice.hpp"
+
+namespace ttnn::operations::data_movement::detail {
+namespace py = pybind11;
+
+void bind_slice(py::module& module) {
+    auto doc =
+        R"doc(
+            slice(input_tensor: ttnn.Tensor, output_tensor_start: ttnn.Shape, output_tensor_end: ttnn.Shape,  value: Union[int, float], *, Optional[ttnn.MemoryConfig] = None) -> ttnn.Tensor
+
+            Returns a sliced tensor.
+
+            Equivalent pytorch code:
+
+            .. code-block:: python
+
+                output_tensor = input_tensor[output_start: output_end]
+
+            Args:
+                * :attr:`input_tensor`: Input Tensor.
+                * :attr:`output_tensor_start`: Shape describing where to start slice.
+                * :attr:`output_tensor_end`: Shape describing where to end slice.
```

Comment:
should these be renamed to `slice_start` and `slice_end` or is it aligned with torch?